oracle-plugin/src/e2e-test/features/sink/OracleDesignTime.feature

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Oracle
+Feature: Oracle sink - Verify Oracle sink plugin design time scenarios
+  @Oracle_Required
```

Comment:
1. We should mention source/sink via tags. Eg. @Oracle @Source
2. What is the significance of: @Oracle_Required ? Are we following some process here?

Reply:
Just added as it was mentioned in existing. Not required hence removed.